addon/objects/uml-primitives/fd-uml-signal-send.js

```diff
@@ -0,0 +1,81 @@
+/**
+  @module ember-flexberry-designer
+*/
+
+import Ember from 'ember';
+
+import { SignalReceiptRight } from './fd-uml-signal-receipt';
+import FdUmlElement from './fd-uml-element';
+
+/**
+  An object that describes a SignalSend elements on the UML diagram.
+
+  @class FdUmlSignalSend
+  @extends FdUmlElement
+*/
+export default FdUmlElement.extend({
+
+  /**
+    The name of the ActiveObject.
+
+    @property name
+    @type String
+  */
+  name: Ember.computed.alias('primitive.Name.Text'),
+
+  /**
+    Type of primitive.
+
+    @property type
+    @type String
+  */
+  type: Ember.computed.alias('primitive.$type'),
+
+  /**
+    See {{#crossLink "FdUmlPrimitive/JointJS:method"}}here{{/crossLink}}.
+
+    @method JointJS
+  */
+  JointJS() {
+    let properties = this.getProperties('id', 'name', 'size', 'position', 'type');
+    if (properties.type === 'STORMCASE.UML.ad.SignalSendLeft, UMLAD') {
+      return new SignalSendLeft(properties);
+    } else {
+      return new SignalSendRight(properties);
+    }
+  },
+});
+
+/**
+  Defines the JointJS object, which represents a `SignalSendRight` figure.
+
+  @for FdUmlSignalSend
+  @class SignalSendRight
+  @extends flexberry.uml.SignalReceiptRight
+  @namespace flexberry.uml
+  @constructor
+*/
+export let SignalSendRight = SignalReceiptRight.define('flexberry.uml.SignalSendingRight', {
```

Comment:
Use the same name for the exported objects and class names in the `JointJS` namespace, alos [here](https://github.com/Flexberry/ember-flexberry-designer/pull/189/files#diff-8ee60ff7cd0c76a16344ba45a81da637R75).